test/unit/spec/function_output/kbaseDefaultObjectView-spec.js

```diff
@@ -57,34 +58,26 @@ define([
                 size = 1234567,
                 upa = String(ws) + '/' + String(oid) + '/' + String(ver);
 
-            jasmine.Ajax.stubRequest('https://ci.kbase.us/services/ws').andReturn({
```

Comment:
Ew. Oops. That was failing because calls were being made to production. Use the Config module instead!